xl_auth/user/models.py

```diff
@@ -158,6 +159,43 @@ def __repr__(self):
         return '<PasswordReset({email!r})>'.format(email=self.user.email)
 
 
+class LoginAttempt(SurrogatePK, Model):
+    """A failed login attempt."""
+
+    __tablename__ = 'login_attempts'
+    username = Column(db.String(255), unique=False, nullable=False, index=True)
+    source_ip = Column(db.String(255), unique=False, nullable=False, index=True)
```

Comment:
I think this should be renamed to `remote_addr`. This is the term used in the Werkzeug/Flask domain. It's more readable if we use the same word for the same thing, and use different words for different things!

Reply:
Renamed.